### PR TITLE
BA125 validation

### PR DESCRIPTION
**Pull request by yedidyacohenpalo**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9997

### CODE REVIEW

File: demisto_sdk/commands/validate/tests/test_tools.py [hunk 1]
```diff
@@ -201,9 +207,14 @@ def create_script_object(
...
     pack = REPO.create_pack()
+    additional_params = {}
     if pack_info:
         pack.set_data(**pack_info)
-    script = pack.create_script(yml=yml_content)
+    if readme_content is not None:
+        additional_params["readme"] = readme_content
+
+    script = pack.create_script(yml=yml_content, **additional_params )
+    script.code.write("from MicrosoftApiModule import *")
```
Comment (jbabazadeh) [inline demisto_sdk/commands/validate/tests/test_tools.py:217]: You already have this line.

File: demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            "long-running-execution",
+        ]
+    )
+
+
+class CustomerFacingDocsDisallowedTermsValidator(BaseValidator[ContentTypes]):
+    error_code = "BA125"
+    description = "Validate that customer facing docs and fields don't contain any internal terms that aren't clear for customers."
+    rationale = "Ensure customer-facing docs avoid internal terms for clarity."
+    error_message = "Found internal terms in a customer-facing documentation file: {terms}"
```
Comment (jbabazadeh) [inline demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py:33]: add to the error message in which file you found the terms
```suggestion
    error_message = "Found internal terms : {terms} in a customer-facing documentation file: {files}"
```

**Review by jbabazadeh — COMMENTED** [review]
Great work! 
a few comments.

File: demisto_sdk/commands/validate/tests/test_tools.py [hunk 3]
```diff
@@ -75,12 +76,16 @@ def create_integration_object(
...
     yml_content = load_yaml("integration.yml")
     update_keys(yml_content, paths, values)
+
     pack = REPO.create_pack()
     if pack_info:
         pack.set_data(**pack_info)
 
     additional_params = {}
 
+    if description_content is not None:
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/test_tools.py:86]:
```suggestion
    if description_content:
```

File: demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    def get_related_files(self, content_item):
+        if isinstance(content_item, Integration):
+            return [content_item.readme, content_item.description_file]
+        elif isinstance(content_item, Pack):
+            return [content_item.readme, content_item.release_note]
+        elif isinstance(content_item, (Playbook, Script)):
+            return [
+                content_item.readme,
+            ]
+        raise ValueError(f"unexpected content item type {content_item.content_type}")
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py:83]:
```suggestion
        related_files = [content_item.readme]
        if isinstance(content_item, Integration):
            related_files.append(content_item.description_file)
        elif isinstance(content_item, Pack):
            related_files.append(content_item.release_note)
        return related_files
```

File: demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+            for content_item in content_items
+            if self.find_disallowed_terms(
+                self.get_related_files(content_item), found_terms
+            )
+        ]
+
+    def format_error_message(self, found_terms):
+        return ", ".join(
+            f"{', '.join(found_terms[file])} in {file}" for file in found_terms
+        )
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py:60]:
```suggestion
    def format_error_message(self, found_terms):
        return "\n".join(
            f"{', '.join(found_terms[file])} in {file}" for file in found_terms
        )
```
Let's split the different files with \n

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Looks good!
Minor comments.

File: demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py (hunk 2 above)
Comment (yedidyacohenpalo) [reply demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py]: done

**Review by jbabazadeh — APPROVED** [review]
nice!

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**